include/bluetooth/gatt.h

```diff
@@ -1208,10 +1208,30 @@ struct bt_gatt_exchange_params {
  *
  *  @note Shall only be used once per connection.
  *
+ *  The Response comes in callback @p params->func. The callback is run from
+ *  the Bluetooth event context. @p *params must remain valid until start of
```

Comment:
"Bluetooth event context" is not described or mentioned anywhere else. I suppose you mean the RX thread?

Reply:
```suggestion
 *  the Bluetooth event context. @p params must remain valid until start of
```